src/pages/users/users-details/users-details.js

```diff
@@ -0,0 +1,222 @@
+
+  const {
+    firstName,
+    lastName,
+    country,
+    city,
+    adress,
+    postCode,
+    ban
+  } = useUsersHandler(id);
+
+  const status = ban ? USER_UNACTIVE_STATUS : USER_ACTIVE_STATUS;
+  const buttonStatus = ban ? USER_ACTIVE_TITLE : USER_UNACTIVE_TITLE;
+
+  const userStatusHandler = (userId) => {
+    const updateStatus = () => {
+      dispatch(closeDialog());
+      dispatch(updateUserStatus(userId));
+    };
+    openSuccessSnackbar(
+      updateStatus,
+      SWITCH_USER_STATUS_TITLE,
+      SWITCH_USER_STATUS_MESSAGE,
+      SWITCH_USER_STATUS_TITLE
+    );
+  };
+
+  if (loading) {
+    return <LoadingBar />;
+  }
+
+  return (
+    <div className={styles.detailsContainer}>
+      <form>
+        <FormControl className={styles.userDetails}>
+          <Grid container spacing={1}>
+            <Grid item xs={12}>
+              <Typography variant='h1' className={styles.detailsTitle}>
+                Інформація про користувача
+              </Typography>
+            </Grid>
+            <Grid item xs={12}>
+              <Paper className={styles.userBlock}>
+                <Grid container spacing={1} className={styles.userHeader}>
+                  <Grid item xs={1} className={styles.userAvatar}>
+                    <Avatar>{`${firstName[0]}${lastName[0]}`}</Avatar>
+                  </Grid>
+                  <Grid item xs={2}>
+                    <Typography className={styles.userName}>
+                      {`${firstName} ${lastName}`}
+                    </Typography>
+                    <Typography className={styles.userStatus}>
+                      {status}
+                    </Typography>
+                  </Grid>
+                </Grid>
+              </Paper>
+            </Grid>
+            <Grid item xs={12}>
+              <Paper className={styles.userBlock}>
+                <TextField
+                  id='firstName'
+                  className={styles.textField}
+                  variant='outlined'
+                  label="Ім'я"
+                  value={firstName}
+                  InputLabelProps={{
+                    classes: {
+                      root: styles.inputLabel,
+                      shrink: 'shrink'
+                    }
+                  }}
+                  multiline
+                  disabled
+                />
+                <TextField
+                  id='lastName'
+                  className={styles.textField}
+                  variant='outlined'
+                  label='Прізвище'
+                  value={lastName}
+                  InputLabelProps={{
+                    classes: {
+                      root: styles.inputLabel,
+                      shrink: 'shrink'
+                    }
+                  }}
+                  multiline
+                  disabled
+                />
+              </Paper>
+            </Grid>
+            <Grid item xs={12}>
+              <Paper className={styles.userBlock}>
```

Comment:
It is better to move this `Paper` component and its inner components to the separate component to simplify the understanding `users-details` component

Reply:
had fixed that and created new components